client/ayon_houdini/plugins/publish/extract_render.py

```diff
@@ -81,7 +89,8 @@ def process(self, instance):
             if not os.path.exists(frame)
         ]
         if missing_frames:
-            # TODO: Use user friendly error reporting.
-            raise RuntimeError("Failed to complete render extraction. "
-                               "Missing output files: {}".format(
-                                   missing_frames))
+            missing_frames = "\n\n ●  ".join(missing_frames)
```

Comment:
Please use regular dash?
```suggestion
            missing_frames = "\n\n -  ".join(missing_frames)
```

Reply:
You don't like my bullet point 😢. 
The publisher UI doesn't convert `-` to a `●` like we have in markdown.

Reply:
https://github.com/ynput/ayon-houdini/pull/190/commits/bde4ab355fe5a33651227f5557a66fe81a4b2fef